config/locales/gobierto_dashboards/views/ca.yml

```diff
@@ -6,18 +6,34 @@ ca:
         assignee: Adjudicatari
         contract_amount: Imp. adjudicació
         contractor: Contractor
-        contracts: Contractes
         description: Visualització i anàlisi dels contractes adjudicats i les licitacions
           del %{entity_name}
         empty_table: No hi ha dades disponibles
         end_date: Data
         final_amount: Quantitat
+        nav:
+          contracts: Contracts
+          summary: Resum
+          tenders: Licitacions
         process_type: Tipus
         status: Procediment
         submission_date: Data
-        summary: Resum
+        summary:
+          contracts: Contractes adjudicats
+          contracts_for: contractes per import de
+          label_half_spendings_contracts: El <strong> <span id = "half-spendings-contracts-pct">
+            </ span> </ strong> de contractes concentren el 50% de tota la despesa
+          label_larger_contract_amount: El major contracte suposa un <strong> <span
+            id = "larger-contract-amount-pct"> </ span> </ strong> de tota la despesa
+            en contractes
+          label_less_than_1000: El <strong><span id="less-than-1000-pct"></span></strong>
```

Comment:
HTML in a translation block?

Reply:
Yeah, I thought about this one for a while. I've just realized that in Budget there's a suffix "_html" in the key for this kind of translations.  Is there a better way to approach this problem?

Reply:
I've seen the file. This way is quite a lot ofuscated, as you're adding an `id` to the block which you'll gonna update that value later with a `document.getElementById`, is that right?
This is barely legible.
I would split the translations into smaller parts, to leave here just plain text

Reply:
So, spliting in smaller parts would make these translation keys. There are quotation marks because otherwise it'd be easy to miss the required whitespaces (although it can also be fixed through js).

```yaml 
label_half_spendings_contracts_1: "El "
label_half_spendings_contracts_2: " de contratos concentran el 50% de todo el gasto"
```

Later in vue I'd need to do this (triming just in case). 

```javascript
let arrayHalfSpendingsContracts = [
  I18n.t('gobierto_dashboards.dashboards.contracts.summary.label_half_spendings_contracts_1'),
  '<strong><span id="half-spendings-contracts-pct"></span></strong>',
  I18n.t('gobierto_dashboards.dashboards.contracts.summary.label_half_spendings_contracts_2')
]

this.labelHalfSpendingsContracts = arrayHalfSpendingsContracts.map((str) => str.trim()).join(' ')
```

What do you think? All in all, I'm not entirely sure this adds more clarity.

Reply:
No, the goal is avoid messing with the logic/stuff. HTML in the template tag, javascript in the script tag, translation in the yml, and so on.
So this array is also mixing unalike stuff